setup.sh

```diff
@@ -0,0 +1,4 @@
+#python dependencies required for the tool
```

Comment:
We can make a requirements file for this. For example: https://github.com/mapbox/wikimama/blob/master/requirements.txt